src/game/features/network/QuitSession.cpp

```diff
@@ -0,0 +1,20 @@
+#include "core/commands/Command.hpp"
+#include "game/rdr/Natives.hpp"
+#include "game/features/Features.hpp"
+#include "core/frontend/Notifications.hpp"
+
+namespace YimMenu::Features
+{
+	class QuitSession : public Command
+	{
+		using Command::Command;
+
+		virtual void OnCall() override
```

Comment:
Can be removed if its already added, pr is 3 months old, this would have been here first lol

Reply:
We have `SCRIPTS::BAIL_TO_LANDING_PAGE` and not `NETWORK::NETWORK_SESSION_LEAVE_SESSION()`. I don't know if there is a huge difference, if any at all but it could be considered.